cylc/flow/scheduler.py

```diff
@@ -1844,35 +1844,39 @@ def _check_startup_opts(self) -> None:
         """Abort if "cylc play" options are not consistent with type of start.
 
         * Start from cycle point or task is not valid for a restart.
-        * Ignore initial point (etc.) is not valid for a new run.
+        * Reloading of cycle points is not valid for a new run.
         """
-        if self.is_restart:
-            for opt in ('icp', 'startcp'):
-                if getattr(self.options, opt, None) not in (None, 'ignore'):
-                    raise SchedulerError(
-                        f"option --{opt} is not valid for restart.")
-            opt = 'starttask'
-            if getattr(self.options, opt, None) is not None:
-                raise SchedulerError(
-                    f"option --{opt} is not valid for restart.")
-        else:
-            for opt in ('icp', 'fcp', 'startcp', 'stopcp', 'starttask'):
-                if getattr(self.options, opt, None) == 'ignore':
-                    raise SchedulerError(
-                        f"option --{opt}=ignore is only valid for restart.")
+        for opt in ('icp', 'startcp', 'starttask'):
+            value = getattr(self.options, opt, None)
+            if self.is_restart:
+                if value is not None:
+                    raise UserInputError(
```

Comment:
Checked that this change of exception doesn't mess with the error catching.

Works ok, changes the exit code from `1` to `3` but that doesn't matter.

Error message changes from:

```
2021-08-05T10:39:28+01:00 ERROR - Workflow shutting down - option --startcp=reload is not valid (only --fcp and --stopcp can be 'reload')
```

To:

```
2021-08-05T10:38:53+01:00 ERROR - Workflow shutting down - UserInputError: option --startcp=reload is not valid 
```